stdlib/2and3/sqlite3/dbapi2.pyi

```diff
@@ -26,7 +26,7 @@ if sys.version_info >= (3,):
 else:
     Binary = buffer
 
-def register_adapters_and_converters(): ...
+def register_adapters_and_converters() -> None: ...
```

Comment:
This function actually should be removed from the stub, since it's `del`ed in the code: https://github.com/python/cpython/blob/master/Lib/sqlite3/dbapi2.py#L89